rules/dotnet-upgrade-for-complex-projects/rule.md

```diff
@@ -99,61 +99,74 @@ There are several ways to migrate project from ASP.NET to ASP.NET Core. We stron
 
 ### Create side-by-side incremental project with [.NET Upgrade Assistant](https://dotnet.microsoft.com/en-us/platform/upgrade-assistant)
 
-This will create a new .NET 8 project. For functionalities that have not yet been migrated, YARP will redirect them to the .NET Framework web application.
+After you've [installed the .NET Upgrade Assistant extension](https://learn.microsoft.com/en-au/dotnet/core/porting/upgrade-assistant-install#install-the-visual-studio-extension),
+right-click on the project in the Solution Explorer window, and select Upgrade.
 
-### Configure YARP
+![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/3699937/3303daaf-0dea-4b34-9f59-53fd55acf2ef)
+
+A tab is opened which provides, based on your project type, different styles of upgrade:
+
+- In-place project upgrade
+  This option upgrades your project without making a copy.
+
+- Side-by-side project upgrade
+  Copies your project and upgrades the copy, leaving your original project alone.
+
+- Side-by-side incremental
+  A good choice for complicated web apps. Upgrading from ASP.NET to ASP.NET Core requires quite a bit of work and at times manual refactoring. This mode puts a .NET project next to your existing .NET Framework project, and routes endpoints that are implemented in the .NET project, while all other calls are sent to .NET Framework application.
+
+  This mode lets you slowly upgrade your ASP.NET or Library app piece-by-piece.
+
+On more complex projects you might find that Upgrade Assistant only provides you with the side-by-side incremental option. That is also the option we will be focusing on.
+
+Once your app has been upgraded, a status screen is displayed which shows all of the artifacts related to your project that were associated with the upgrade. Each upgrade artifact can be expanded to read more information about the status. The following list describes the status icons:
+
+- **Filled green checkmark**: The artifact was upgraded and completed successfully.
+- **Unfilled green checkmark**: The tool didn't find anything about the artifact to upgrade.
+- **Yellow warning sign**: The artifact was upgraded, but there's important information you should consider.
+- **Red X**: The artifact was to be upgraded, but the upgrade failed.
+  ![Figure: Example results from an Upgrade Assistant upgrade.](https://github.com/SSWConsulting/SSW.Rules.Content/assets/3699937/8dc7aaac-90cd-425b-bea6-5c74bea1ad57)
+
+
+After the Upgrade Assistant completes the upgrade you will find a new project in the solution running .NET 8. Upgrade Assistant pre-configured the project and installed any necessary packages it required to run side-by-side using the Yarp proxy. 
+
+### Configure Yarp
+
+ ![Figure: The new project architecture and request flow.](https://github.com/SSWConsulting/SSW.Rules.Content/assets/3699937/36e807f5-e7e8-475c-8a82-d1b972eff35e)

```

Comment:
The text ".NET 8" and ".NET Framework" are misaligned with their respective blocks.